### ISSUE TITLE
Implement Template team's Templates in PDF

### ISSUE THREAD

**Issue by MrL9000** [uprm-inso4115-2023-2024-s2/semester-project-automated-resume-builder#121]
Communicate with the template team to implement the templates they have created for the PDF converter to use. With PDFKit already set up, the project already has the functionality to build and export the projects in a PDF format, and there is an entire team dedicated to the development of the templates that users will use to create their resumes. All that would be missing is taking the templates the template team made, and implement them in PDFKit to be able to download the templates the users filled out.

**Comment by irsaris** [comment]
Hello, please suggest urgency and difficulty scores.

**Comment by irsaris** [comment]
Please thoroughly explain this task and suggest scoring.

**Comment by MrL9000** [comment]
I'm not too sure how to handle the numbering, but I personally think this will be a relatively tough task due to there being multiple different templates and the possibility that there might be a decent amount of extra formatting per template, which might require a little more back end development depending on the level of complexity the templates may have. There's also the fact that I've yet to have the chance to communicate with the template team, so I'm not sure what will be the number and complexity of these templates. Lastly, Due to the fact I'm intending to make this my main (and possibly only) task, I'd recommend a score that is enough to cover most if not all the scoring required for the semester, so something like an urgency of 5 and a difficulty of 4, up to you guys to decide on the particulars.

TL;DR:
Urgency 5
Diffuclty 4
(maybe because it's hard to guage the difficulty without knowing what are the templates)

**Comment by MrL9000** [comment]
This task has to be subdivided into multiple tasks due to the Front-End Back-End Conflict.
